packages/global-styles/src/components/color-palette-variations/preview.tsx

```diff
@@ -60,54 +39,26 @@ const ColorPaletteVariationPreview = ( { title }: Props ) => {
 						overflow: 'hidden',
 					} }
 				>
-					{ title ? (
-						<HStack
-							spacing={ 10 * ratio }
-							justify="center"
-							style={ {
-								height: '100%',
-								overflow: 'hidden',
-							} }
-						>
-							{ highlightedColors.map( ( color, index ) => (
-								<div
-									key={ index }
-									style={ {
-										height: normalizedSwatchSize,
-										width: normalizedSwatchSize,
-										background: color,
-										borderRadius: normalizedSwatchSize / 2,
-									} }
-								/>
-							) ) }
-						</HStack>
-					) : (
-						<VStack
-							spacing={ 3 * ratio }
-							justify="center"
-							style={ {
-								height: '100%',
-								overflow: 'hidden',
-								padding: 10 * ratio,
-								boxSizing: 'border-box',
-							} }
-						>
+					<HStack
+						spacing={ 10 * ratio }
+						justify="center"
+						style={ {
+							height: '100%',
+							overflow: 'hidden',
+						} }
+					>
+						{ [ textColor, buttonColor ].map( ( color, index ) => (
 							<div
+								key={ index }
 								style={ {
-									fontSize: 40 * ratio,
-									fontFamily: headingFontFamily,
-									color: headingColor,
-									fontWeight: headingFontWeight,
-									lineHeight: '1em',
-									textAlign: 'center',
+									height: normalizedSwatchSize,
+									width: normalizedSwatchSize,
+									background: color,
+									borderRadius: normalizedSwatchSize / 2,
 								} }
-							>
-								{ translate( 'Default', {
```

Comment:
I removed this whole logic because we no longer show this `Default` text in color swatches.